src/Components/Filter/PriceRange.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import "./style.css"
+import { useState } from 'react';
```

Comment:
not necessary but you can do import like this import React,{ useState } from 'react';

Reply:
Changed it